packages/types/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { BigNumber } from '@0xproject/utils';
+import { BigNumber } from 'bignumber.js';
```

Comment:
Why this change?

Reply:
Because here we needed only the type and it helped to break a circular dependency.